components/company/EditForm.vue

```diff
@@ -0,0 +1,370 @@
+    {{ $t(`${alertState.alertRoute}`) }}
+
+    <template #actions>
+      <VBtn
+        :color="alertState.type"
+        variant="text"
+        @click="alertState.show = false"
+      >
+        {{ $t('alert.close_alert') }}
+      </VBtn>
+    </template>
+  </VSnackbar>
+
+  <!-- Edit Form -->
+  <VForm ref="form" @submit.prevent="saveChanges || createEvent">
+    <VContainer>
+      <!-- Name -->
+      <VRow>
+        <FormTextInput
+          v-model="state.name"
+          :content="{
+            label: $t('edit.company.attributes.name'),
+          }"
+          :rules="[useRequiredInput]"
+        />
+      </VRow>
+
+      <!-- Description -->
+      <VRow>
+        <FormTextareaInput
+          v-model="state.description"
+          :content="{
+            label: $t('edit.company.attributes.description'),
+          }"
+        />
+      </VRow>
+
+      <!-- Company Type -->
+      <VRow>
+        <FormSelectInput
+          v-model="state.type"
+          :content="{
+            label: $t('edit.company.attributes.type.name'),
+            options: [
+              {
+                title: $t('edit.company.attributes.type.main_partner'),
+                value: 'main-partner',
+              },
+              {
+                title: $t('edit.company.attributes.type.partner'),
+                value: 'partner',
+              },
+              {
+                title: $t('edit.company.attributes.type.sponsor'),
+                value: 'sponsor',
+              },
+            ],
+          }"
+          :rules="[useRequiredInput]"
+        />
+      </VRow>
+
+      <!-- Website -->
+      <VRow>
+        <FormTextInput
+          v-model="state.webpage"
+          :content="{
+            label: $t('edit.company.attributes.website'),
+          }"
+          :rules="[useRequiredInput]"
+        />
+      </VRow>
+
+      <!-- Logo -->
+      <!-- TODO: #190 Add code for uploading image after company creation -->
+      <VRow>
+        <VCol :cols="state.logo ? 6 : 8">
+          <FormFileInput
+            :content="{
+              label: $t('edit.company.attributes.logo'),
+              icon: 'mdi-image',
+              accept: ['image/png', 'image/jpeg'],
+            }"
+            :rules="[useRequiredInput]"
+            @change="setLogo"
+          />
+        </VCol>
+        <VCol v-if="state.logo" cols="2">
+          <VTooltip location="top" color="primary">
+            <!-- eslint-disable-next-line -->
+            <template #activator="{ props }">
+              <VBtn
+                v-bind="props"
+                variant="flat"
+                block
+                size="large"
+                rounded="lg"
+                color="error"
+                icon="mdi-delete"
+                @click="deleteLogo"
+              ></VBtn>
+            </template>
+
+            {{ $t('edit.company.delete_logo') }}
+          </VTooltip>
+        </VCol>
+        <VCol cols="4">
+          <VBtn
```

Comment:
I don't really understand why theres a separate button for updating the logo. I think this should be moved to the _save changes_ button.